distributed/comm/tests/test_ucx.py

```diff
@@ -169,22 +171,13 @@ def test_ucx_deserialize():
         lambda cudf: cudf.DataFrame([1]).head(0),
         lambda cudf: cudf.DataFrame([1.0]).head(0),
         lambda cudf: cudf.DataFrame({"a": []}),
-        pytest.param(
-            lambda cudf: cudf.DataFrame({"a": ["a"]}).head(0),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
-        ),
-        pytest.param(
-            lambda cudf: cudf.DataFrame({"a": [1.0]}).head(0),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
-        ),
-        pytest.param(
-            lambda cudf: cudf.DataFrame({"a": [1]}).head(0),
-            marks=pytest.mark.xfail(reason="0 length objects don't deseralize cleanly"),
-        ),
+        lambda cudf: cudf.DataFrame({"a": ["a"]}).head(0),
+        lambda cudf: cudf.DataFrame({"a": [1.0]}).head(0),
+        lambda cudf: cudf.DataFrame({"a": [1]}).head(0),
```

Comment:
All of these worked fine. Not sure if there is something I'm missing though.

Reply:
The used to not.  Thanks for adding this back in